code/modules/mob/dead/new_player/sprite_accessories.dm

```diff
@@ -1504,3 +1505,216 @@
+	name = "Square wave"
+	icon_state = "squarewave"
+
+datum/sprite_accessory/ipc_screens/stars
+	name = "Stars"
+	icon_state = "stars"
+
+/datum/sprite_accessory/ipc_screens/static_screen
+	name = "Static"
+	icon_state = "static"
+
+datum/sprite_accessory/ipc_screens/textdrop
+	name = "Text Drop"
+	icon_state = "textdrop"
+
+/datum/sprite_accessory/ipc_screens/yellow
+	name = "Yellow"
+	icon_state = "yellow"
+
+/datum/sprite_accessory/ipc_antennas
+	icon = 'icons/mob/ipc_accessories.dmi'
+	color_src = HAIR
+
+/datum/sprite_accessory/ipc_antennas/none
+	name = "None"
+	icon_state = "None"
+
+/datum/sprite_accessory/ipc_antennas/angled
+	name = "Angled"
+	icon_state = "antennae"
+
+/datum/sprite_accessory/ipc_antennas/antlers
+	name = "Antlers"
+	icon_state = "antlers"
+
+/datum/sprite_accessory/ipc_antennas/crowned
+	name = "Crowned"
+	icon_state = "crowned"
+
+/datum/sprite_accessory/ipc_antennas/cyberhead
+	name = "Cyberhead"
+	icon_state = "cyberhead"
+
+/datum/sprite_accessory/ipc_antennas/droneeyes
+	name = "Drone Eyes"
+	icon_state = "droneeyes"
+
+/datum/sprite_accessory/ipc_antennas/light
+	name = "Light"
+	icon_state = "light"
+
+/datum/sprite_accessory/ipc_antennas/sidelights
+	name = "Sidelights"
+	icon_state = "sidelights"
+
+/datum/sprite_accessory/ipc_antennas/tesla
+	name = "Tesla"
+	icon_state = "tesla"
+
+/datum/sprite_accessory/ipc_antennas/tv
+	name = "TV Antenna"
+	icon_state = "tvantennae"
+
+/datum/sprite_accessory/ipc_chassis // Used for changing limb icons, doesn't need to hold the actual icon. That's handled in ipc.dm
+	icon = null
+	icon_state = "who cares fuck you" // In order to pull the chassis correctly, we need AN icon_state(see line 36-39). It doesn't have to be useful, because it isn't used.
+	color_src = 0
+
+/datum/sprite_accessory/ipc_chassis/mcgreyscale
+	name = "Morpheus Cyberkinetics(Greyscale)"
+	limbs_id = "mcgipc"
+	color_src = MUTCOLORS
+
+/datum/sprite_accessory/ipc_chassis/bishopcyberkinetics
+	name = "Bishop Cyberkinetics"
+	limbs_id = "bshipc"
+
+/datum/sprite_accessory/ipc_chassis/bishopcyberkinetics2
+	name = "Bishop Cyberkinetics 2.0"
+	limbs_id = "bs2ipc"
+
+/datum/sprite_accessory/ipc_chassis/hephaestussindustries
+	name = "Hephaestus Industries"
+	limbs_id = "hsiipc"
+
+/datum/sprite_accessory/ipc_chassis/hephaestussindustries2
+	name = "Hephaestus Industries 2.0"
+	limbs_id = "hi2ipc"
+
+/datum/sprite_accessory/ipc_chassis/shellguardmunitions
+	name = "Shellguard Munitions Standard Series"
+	limbs_id = "sgmipc"
+
+/datum/sprite_accessory/ipc_chassis/wardtakahashimanufacturing
+	name = "Ward-Takahashi Manufacturing"
+	limbs_id = "wtmipc"
+
+/datum/sprite_accessory/ipc_chassis/xionmanufacturinggroup
+	name = "Xion Manufacturing Group"
+	limbs_id = "xmgipc"
+
+/datum/sprite_accessory/ipc_chassis/xionmanufacturinggroup2
+	name = "Xion Manufacturing Group 2.0"
+	limbs_id = "xm2ipc"
+
+/datum/sprite_accessory/ipc_chassis/zenghupharmaceuticals
+	name = "Zeng-Hu Pharmaceuticals"

```

Comment:
this is a bunch of deep lore shit that we dont have, doesnt matter particularly

Reply:
The different corporation names just designate different sprite styles. Most of them are assigned by the creators of the sprites themselves (Shellguard, for example.)

Reply:
It really doesn't conflict with our lore, and it doesn't hurt to have some extra loose ends hanging around that others can pick up if appropriate for their own PRs. 👍

Reply:
Whereas Moths have the wing style name, humans have hail styles and stuff, it makes sense for a bunch of previously lawed servitors to have brands for their part names.